modules/audio/inference/direction_detection.h

```diff
@@ -0,0 +1,42 @@
+
+/******************************************************************************
+ * Copyright 2020 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#pragma once
+
+#include <algorithm>
+#include <cmath>
+#include <vector>
+
+#include "ATen/ATen.h"
+#include "cyber/cyber.h"
+#include "torch/torch.h"
+
+namespace apollo {
+namespace audio {
+
+constexpr float SOUND_SPEED = 343.2;
+constexpr float MIC_DISTANCE_4 = 0.08127;
+constexpr float MAX_TDOA_4 = MIC_DISTANCE_4 / SOUND_SPEED;
+constexpr float PI = 3.1415926535897932384626433832795;
```

Comment:
Please use cmath pi instead of user defined.

Reply:
Sure